### DISCUSSION TITLE
快捷操作反向代理配置后Host被重置

### DISCUSSION THREAD

**Discussion by 0xddy** [1Panel-dev/1Panel#5318] [Q&A]
### 1Panel 版本

v1.10.9-lts

### 问题描述

比如在反向代理页面，点击开启关闭代理缓存开关后，Host 配置会被重置。能否人性化一些，快捷设置的时候，源文件多加一些读取和判断。还有就是面板上建议显示 nginx 内部IP这样反代本地方便很多，默认 127.0.0.1 是不通的

### 重现步骤

比如在反向代理页面，点击开启关闭代理缓存开关

**Comment by zhengkunwang223** [comment]
有具体的配置吗？前后配置对比 
我刚才试了一下 proxy_set_header Host 这个是不会被重置的

**Comment by 0xddy** [comment]
@zhengkunwang223  可能我没注意 修改后端的输入框失去焦点的时候，Host 输入框会重置.... 直接点了确定保存，这个也有点操作体验上小问题，要不不变动，要么实时变动有个提醒
